.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
We don't need this file

Reply:
Add to gitignore and check that in

Reply:
I don't understand what you are saying. This is the gitignore!